doc/source/whatsnew/v1.1.2.rst

```diff
@@ -30,7 +30,7 @@ Bug fixes
 - Bug in :class:`Series` constructor raising a ``TypeError`` when constructing sparse datetime64 dtypes (:issue:`35762`)
 - Bug in :meth:`DataFrame.apply` with ``result_type="reduce"`` returning with incorrect index (:issue:`35683`)
 - Bug in :meth:`DateTimeIndex.format` and :meth:`PeriodIndex.format` with ``name=True`` setting the first item to ``"None"`` where it should bw ``""`` (:issue:`35712`)
--
+- Bug in :meth:`PeriodIndex.get_loc` incorrectly raising ``ValueError`` on non-datelike strings instead of ``KeyError``, causing similar errors in :meth:`Series.__geitem__`, :meth:`Series.__contains__`, and :meth:`Series.loc.__getitem__` (:issue:`34240`)
```

Comment:
hmm, is this needed to do in 1.1.2?

Reply:
or i guess we are treating as a regression?

Reply:
i think its a bugfix, not regression.  im fine putting it in 1.2

Reply:
ok, great, ping on green.

Reply:
ping

Reply:
can u move to 1.2

Reply:
moved+green